src/glwidget.h

```diff
@@ -97,9 +97,9 @@ public slots:
 
     int mRID = 0;
     int mAID = 0;
-    int mOx = 0;
-    int mOy = 0;
-    int mOz = 0;
+    int mMapCenterX = 0;
+    int mMapCenterY = 0;
+    int mMapZLevel = 0;
```

Comment:
Although it does represent a "level" it is functionally the same as the other two (though we only use integer values for it) so it should have the same type of name
```suggestion
    int mMapCenterZ = 0;
```